src/api/todos.ts

```diff
@@ -5,4 +5,12 @@ export const getTodos = (userId: number) => {
   return client.get<Todo[]>(`/todos?userId=${userId}`);
 };
 
-// Add more methods here
+export const postTodo = (userId: number, title: string) => {
+  const todo = {
+    userId,
+    title,
+    id: new Date(),
```

Comment:
i find `Date.now()` more suitable for an id
compared to `new Date()`

<img width="616" alt="Date now()" src="https://user-images.githubusercontent.com/39308717/219001310-a47872a7-791d-440c-a018-5f4b46752a25.png">

```suggestion
    id: Date.now(),
```